ReSharper.FSharp/src/FSharp.Psi.Features/FSharp.Psi.Features.fsproj

```diff
@@ -1,159 +1,160 @@
-    <Compile Include="src\QuickFixes\RemoveRedundantBackticksFix.fs" />
-    <Compile Include="src\QuickFixes\RemoveInlineFix.fs" />
-    <Compile Include="src\QuickFixes\RemoveUnnecessaryUpcastFix.fs" />
-    <Compile Include="src\QuickFixes\ToRecursiveFunctionFix.fs" />
-    <Compile Include="src\QuickFixes\AddParensFix.fs" />
-    <Compile Include="src\QuickFixes\RemoveSubsequentFix.fs" />
-    <Compile Include="src\QuickFixes\RemoveRedundantNewFix.fs" />
-    <Compile Include="src\QuickFixes\RemoveUnusedLocalBindingFix.fs" />
-    <Compile Include="src\QuickFixes\ReplaceLetWithExpressionFix.fs" />
-    <Compile Include="src\QuickFixes\ConvertToUseFix.fs" />
-    <Compile Include="src\QuickFixes\ToUpcastFix.fs" />
-    <Compile Include="src\QuickFixes\IntroduceVarFix.fs" />
-    <Compile Include="src\QuickFixes\RemoveNeverMatchingRuleFix.fs" />
-    <Compile Include="src\PostfixTemplates\PostfixTemplates.fs" />
-    <Compile Include="src\PostfixTemplates\NotTemplate.fs" />
-    <Compile Include="src\PostfixTemplates\LetPostfixTemplate.fs" />
-    <Compile Include="src\PostfixTemplates\MatchPostfixTemplate.fs" />
-    <Compile Include="src\PostfixTemplates\ForPostfixTemplate.fs" />
-    <Compile Include="src\LanguageService\FSharpDeclaredElementIconProvider.fs" />
-    <Compile Include="src\LanguageService\FSharpPsiProperties.fs" />
-    <Compile Include="src\LanguageService\FSharpProjectFileLanguageService.fs" />
-    <Compile Include="src\LanguageService\FSharpElementFactory.fs" />
-    <Compile Include="src\LanguageService\FSharpLanguageService.fs" />
-    <Compile Include="src\LanguageService\FSharpClrLanguage.fs" />
-    <Compile Include="src\LanguageService\FSharpBreadcrumbsHelper.fs" />
-    <Compile Include="src\LanguageService\FSharpImportTypeHelper.fs" />
-    <Compile Include="src\LanguageService\FSharpQuickDefinitionService.fs" />
-    <Compile Include="src\FSharpTypingAssist.fs" />
-    <Compile Include="src\ZoneMarker.fs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="FSharp.Core" Version="$(FSharpCoreVersion)" />
-    <PackageReference Include="JetBrains.Fantomas" Version="$(FantomasVersion)" />
-    <PackageReference Include="JetBrains.FSharp.Compiler.Service" Version="$(FSharpCompilerServiceVersion)" />
-    <PackageReference Include="JetBrains.Rider.SDK" Version="$(RiderSDKVersion)" />
-    <PackageReference Include="System.ValueTuple" Version="$(ValueTupleVersion)" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\Daemon.FSharp\Daemon.FSharp.csproj" />
-    <ProjectReference Include="..\FSharp.Psi\FSharp.Psi.csproj" />
-    <ProjectReference Include="..\Services.FSharp\Services.FSharp.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Reference Include="PresentationFramework" />
-    <Reference Include="System.Web" />
-    <Reference Include="WindowsBase" />
-    <Reference Include="System.Windows.Forms" />
-  </ItemGroup>
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
What happened to this file?

Reply:
CR→CRLF conversion happened, which is wrong. Other files added by this PR have CRLF, too. E.g. [this one](https://raw.githubusercontent.com/JetBrains/fsharp-support/a5734f1e5819455af07d5890779c44615c82e083/ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/RemoveUnnecessaryArgumentFix.fs).

@DedSec256, please ensure you have set up your Git and/or IDE in the right way (either make the IDE to create files with LF or enable `git.autocrlf` setting).

@auduchinok please make sure no CRLF will slip into our repo from this PR :)